### PR TITLE
Adding Blocknet

### PR DESCRIPTION
**Pull request by sigwo**
### Changelog
* Adding required files for Blocknet addition to Azure BaaS program.

### Description of the change
Files modified for Blocknet template deployment to Azure.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sigwo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by sigwo** [comment]
@singhkay did something change in the requirements for azuredeploy.json? I am using the same file structure as last week but Travis CI and the auto template bot says I have an error with `GEN-UNIQUE-11`. Is this just an auto template bot error? Or something I need to patch up before doing any further PR for other blockchains?

**Comment by singhkay** [comment]
@sigwo Yes, don't worry about these errors from acomghbot. The validate API it calls does some additional checking now for available DNS and storage names.

**Comment by sigwo** [comment]
@singhkay excellent. please excuse my messy pr this time. i'll push the other blockchain from a new branch. :)

**Comment by sigwo** [comment]
@singhkay The CI fails due to the template acomghbot problem, yes?

**Comment by singhkay** [comment]
@sigwo looks like a failure in the script. Can you check? 

```
details": [
      {
        "code": "VMExtensionProvisioningError",
        "message": "VM has reported a failure when processing extension 'newuserscript'. Error message: \"Enable failed: Failed with error: HTTP Error 404: Not Found, Traceback (most recent call last):\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 83, in main\n    daemon(hutil)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 284, in daemon\n    retry_count = download_files_with_retry(hutil, retry_count, wait)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 123, in download_files_with_retry\n    download_files(hutil)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 229, in download_files\n    download_external_files(blob_uris, cmd, hutil)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 430, in download_external_files\n    download_external_file(uri, command, hutil)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 440, in download_external_file\n    download_and_save_file(uri, file_path)\n  File \"/var/lib/waagent/Microsoft.OSTCExtensions.CustomScriptForLinux-1.4.1.0/customscript.py\", line 451, in download_and_save_file\n    src = urllib2.urlopen(uri, timeout=timeout)\n  File \"/usr/lib/python2.7/urllib2.py\", line 127, in urlopen\n    return _opener.open(url, data, timeout)\n  File \"/usr/lib/python2.7/urllib2.py\", line 410, in open\n    response = meth(req, response)\n  File \"/usr/lib/python2.7/urllib2.py\", line 523, in http_response\n    'http', request, response, code, msg, hdrs)\n  File \"/usr/lib/python2.7/urllib2.py\", line 448, in error\n    return self._call_chain(*args)\n  File \"/usr/lib/python2.7/urllib2.py\", line 382, in _call_chain\n    result = func(*args)\n  File \"/usr/lib/python2.7/urllib2.py\", line 531, in http_error_default\n    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)\nHTTPError: HTTP Error 404: Not Found\n\"."
      }
```

**Comment by sigwo** [comment]
Not sure about this error?
```

  1) Template Running 1 Parallel Template Validation(s)... blocknet-blockchain-on-ubuntu/azuredeploy.json & blocknet-blockchain-on-ubuntu/azuredeploy.parameters.json should be valid:
     Error: timeout of 7100000ms exceeded. Ensure the done() callback is being called in this test.
```

**Comment by sigwo** [comment]
@singhkay Everything looks fine on my end. Same template I used from Influx. Do you see anything else I could be missing?

File: bitswift-blockchain-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
+      "type": "string",
+      "defaultValue": "From_Source",
```
Comment (singhkay) [inline bitswift-blockchain-ubuntu/azuredeploy.json]: Is there a PPA to install from?

File: blocknet-blockchain-on-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+    },
+    "dnsLabelPrefix": {
+       "type": "string",
+       "metadata": {
+          "description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+       }
+    },
+    "installMethod": {
+      "type": "string",
+      "defaultValue": "From_Source",
```
Comment (singhkay) [inline blocknet-blockchain-on-ubuntu/azuredeploy.json]: PPA for this?

File: bitswift-blockchain-ubuntu/azuredeploy.json (hunk 1 above)
Comment (sigwo) [inline bitswift-blockchain-ubuntu/azuredeploy.json]: No, not yet.

**Comment by singhkay** [comment]
lgtm, merged. I would suggest adding PPA or binary installs in the next update in addition to building from source

**Pull request merged**